app/controllers/webhooks/webhooks-form.js

```diff
@@ -0,0 +1,103 @@
+const CALLBACK_URL_MAX_LENGTH = 2048
+
+const defaultFieldsSchema = [
+  {
+    id: 'callback_url',
+    valid: [
+      { method: isNotEmpty, message: 'Enter a callback URL' },
+      { method: isValidLength, message: `Callback URL must be ${CALLBACK_URL_MAX_LENGTH} or fewer` }
+    ],
+
+    // https://github.com/alphagov/pay-webhooks/blob/main/src/main/java/uk/gov/pay/webhooks/webhook/exception/WebhooksErrorIdentifier.java
+    errorIdentifiers: {
+      CALLBACK_URL_MALFORMED: 'Callback URL must be a valid URL',
+      CALLBACK_URL_PROTOCOL_NOT_SUPPORTED: 'Callback URL must use the protocol HTTPS',
+      CALLBACK_URL_NOT_ON_ALLOW_LIST: 'Callback URL must be on an approved list of domains for live accounts. PLease contact support'
+    }
+  },
+  {
+    id: 'description'
+  },
+  {
+    id: 'subscriptions',
+    valid: [{ method: isNotEmpty, message: 'Select a payment event' }]
```

Comment:
Since they can select multiple, maybe ‘Select some payment events’?

Reply:
After discussion, this ended up being "Select at least one payment event"